repose-aggregator/extensions/simple-rbac/src/main/resources/META-INF/schema/config/simple-rbac.xsd

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:html="http://www.w3.org/1999/xhtml"
+           xmlns:vc="http://www.w3.org/2007/XMLSchema-versioning"
+           xmlns:xerces="http://xerces.apache.org"
+           xmlns:saxon="http://saxon.sf.net/"
+           xmlns:rbac="http://docs.openrepose.org/repose/simple-rbac/v1.0"
+           xmlns="http://docs.openrepose.org/repose/simple-rbac/v1.0"
+           targetNamespace="http://docs.openrepose.org/repose/simple-rbac/v1.0"
+           elementFormDefault="qualified">
+
+    <!-- Elements -->
+    <xs:element name="simple-rbac" type="SimpleRbacConfig"/>
+
+    <!-- Types -->
+    <xs:complexType name="SimpleRbacConfig">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>The root config type for the Simple RBAC filter configuration file.</html:p>
+            </xs:documentation>
+        </xs:annotation>
+
+        <xs:sequence>
+            <xs:element name="delegating" type="DelegatingType" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="resources" type="xs:string" minOccurs="0" maxOccurs="1">
+                <xs:annotation>
+                    <xs:documentation>
+                        <html:p>
+                            This is a list of resources, HTTP methods, and roles allowed to access them.
+                        </html:p>
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+        </xs:sequence>
+
+        <xs:attribute name="roles-header-name" type="xs:string" use="optional" default="X-ROLES">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        This is the name of the header where that roles are passed in.
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="resources-file-name" type="xs:string" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        This is the name of a file containing a list of resources, HTTP methods, and roles allowed to access them.
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="enable-masking-403s" type="xs:boolean" use="optional" default="false">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Instead of returning a FORBIDDEN (403), return a NOT FOUND (404) or METHOD NOT ALLOWED (405) as appropriate.</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="dot-output" type="xs:string" use="optional">
```

Comment:
The dot is the visualization. It is good for debugging what is going on in API-Checker. It is not produced by default, but essential to figure out what is broke when things go wrong. The WADL option is nice if someone wants to transfer to the full API Validation filter after their API grows beyond what the Simple RBAC filter supports.